### PR TITLE
Fix typo in ch13-01

### PR DESCRIPTION
**Pull request by typebrook**

### CODE REVIEW

File: src/ch13-01-closures.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ Rust 的閉包（closures）是個你能賦值給變數或作為其他函式引
 
 ### 透過閉包建立抽象行為
 
-讓我們處理一個範例是當儲存閉包並在之後才執行是很實況的情況。在過程中，我們會討論到閉包語法、型別推導以及特徵。
+我們的下一個範例-儲存閉包並在之後才執行-是很實用的情況。在過程中，我們會討論到閉包語法、型別推導以及特徵。
```
Comment (weihanglo) [inline src/ch13-01-closures.md]: Recommend to use fullwidth version of "破折號" in Traditional Chinese.

```suggestion
我們的下一個範例──儲存閉包並在之後才執行──是很實用的情況。在過程中，我們會討論到閉包語法、型別推導以及特徵。
```

**Review by weihanglo — COMMENTED** [review]
Hey! Thanks for several fixes. Just a little thing need to tweak.

**Review by weihanglo — APPROVED** [review]
LGTM. Thank you!

File: src/ch13-01-closures.md (hunk 1 above)
Comment (typebrook) [reply src/ch13-01-closures.md]: Of course, changes applied.
I just realized fcitx within cangjie3 need to type `shift + -` to get unicode U+2500.

**Pull request merged**